- Allow custom SDL GetPlatform name with env var SDL_PLATFORM

Index: src/SDL.c
--- src/SDL.c.orig
+++ src/SDL.c
@@ -558,6 +558,10 @@ SDL_GetRevisionNumber(void)
 const char *
 SDL_GetPlatform(void)
 {
+    const char *platform;
+    if (platform = SDL_getenv("SDL_PLATFORM")) {
+        return platform;
+    }
 #if __AIX__
     return "AIX";
 #elif __ANDROID__
